src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.html

```diff
@@ -32,12 +33,139 @@
       </ng-container>
     </cd-card>
   </div>
-
-  <div class="row mx-0">
+  <!-- Inventory Card -->
+  <div class="row mx-0 card-row">
     <cd-card title="Inventory"
              i18n-title
              class="col-sm-3 px-3">
-             Text
+      <ul class="inventory-list">
+        <!-- Hosts -->
+        <li>
+          <div *ngIf="healthData.hosts"
```

Comment:
Thanks for bringing this up, the reason I wrote it this way was because `*ngIf` and `*ngFor` cannot be within the same element and so I thought creating a parent element would fix just that. Though I didn't realize that not having it at the highest level would create this issue